src/test/java/io/pravega/connectors/flink/FlinkPravegaReaderTest.java

```diff
@@ -153,8 +153,9 @@ public void testCancellation() throws Exception {
     private static TestableFlinkPravegaReader<Integer> createReader() {
         ClientConfig clientConfig = ClientConfig.builder().build();
         ReaderGroupConfig rgConfig = ReaderGroupConfig.builder().stream(SAMPLE_STREAM).build();
+        boolean enableMetrics = true;
```

Comment:
Would be nice to see some actual assertions about the metrics being installed, but to obtain the metric group (and thus to make assertions about its contents), one needs a handle on the `operator` (which has `getMetricGroup()`).   A simple fix would be to amend `StreamSourceOperatorTestHarness` (which is our class) to have `getMetricGroup` (since the harness  has an `operator` field).

Reply:
I have added a test case to assert the metric group being created